src/SkylandersTrapTeam/GameMods/EnableDebug/rules.txt

```diff
@@ -0,0 +1,33 @@
+[Definition]
+titleIds = 000500001017C600,0005000010181F00
+name = "Enable Debug"
+path = "Skylanders Trap Team/Game Mods/Enable Debug"

```

Comment:
This should be /Mods/ or /Cheats/ to make it match the other graphic packs. But other then that the PR looks good! Sorry that it took me a while to see this PR :)